frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -360,6 +360,24 @@ export default {
       })
       return applicationItems
     },
+
+    //TODO - Add Supp App PDF function will look very similar, but it will hit a seperate endpoint
+    //the supp app PDF's do not generate yet in Dynamics, so holding off on including that code
+    async downloadPDF(application) {
+      if (application.applicationType === 'OFM') {
```

Comment:
Application type 'OFM' is now used quite a bit on the code. Probably worth a constant.

Reply:
'OFM' application type is also used in CancelApplicationDialog. Can we create a shared constant APPLICATION_TYPES instead?

Reply:
done! I had to rename something in CancelApplication - so you'll see a few more changes made in that file.

Reply:
Makes sense.